app/controllers/application_controller.rb

```diff
@@ -24,6 +24,12 @@ def redirect_after_login
     end
   end
 
+  def redirect_forward
+    path = session[:forward_to]
+    session[:forward_to] = nil
+    redirect_to path
```

Comment:
I'm clearing it out to not break the redirect_back_to that we're using in other places. In most cases, and more globally we're using redirect_back_to, its just for these navbar links that we redirect forward. So the conditional in the redirection method is it checks if a session[:forward_to] exists and uses that if it does.